base/announcement.py

```
@@ -204,24 +240,40 @@ def update_announcement(request, anoun_id):
 
     if request.method == "POST":
         form = AnnouncementForm(request.POST, request.FILES, instance=announcement)
+
         if form.is_valid():
             anou, attachment_ids = form.save(commit=False)
```

Ruff (0.14.14)

[warning] Unpacked variable `attachment_ids` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)